credential/exchange/verification.go

```diff
@@ -30,7 +34,12 @@ func VerifyPresentationSubmission(verifier crypto.JWTVerifier, et EmbedTarget, d
 	}
 	switch et {
 	case JWTVPTarget:
-		_, _, vp, err := signing.VerifyVerifiablePresentationJWT(verifier, string(submission))
+		jwtVerifier, ok := verifier.(crypto.JWTVerifier)
+		if !ok {
+			return fmt.Errorf("verifier<%T> is not a JWT verifier", verifier)
+		}
+		// verify the VP, which in turn verifies all credentials in it
+		_, _, vp, err := credential.VerifyVerifiablePresentationJWT(context.Background(), jwtVerifier, resolver, string(submission))
```

Comment:
eventually context.background should be top level

Reply:
I agree, I will make it top level here